modules/nextflow/src/main/groovy/nextflow/dag/DAG.groovy

```diff
@@ -426,7 +426,7 @@ class DAG {
         /**
          * @return The unique name for this node
          */
-        String getName() { "p${getOrder()}" }
+        String getName() { "v${getOrder()}" }
```

Comment:
Why change this prefix?

Reply:
"v" is for vertex. not sure why it was "p". I guess for process? but then not all vertices are processes

Reply:
Fair enough